README.md

```diff
@@ -458,700 +841,1310 @@ public void Fire()
+
+  
+
+> 4.2.4 [Prefabs](#anc-prefab)
+
+  
+
+> 4.2.5 [Materials](#anc-materials)
+
+  
+
+> 4.2.6 [Textures](#anc-textures)
+
+  
+
+> 4.2.7 [Miscellaneous](#anc-misc)
+
+  
+
+> 4.2.8 [Physics](#anc-physics)
+
+  
+
+> 4.2.9 [Audio](#anc-audio)
+
+  
+
+> 4.2.10 [User Interface](#anc-ui)
+
+  
+
+> 4.2.11 [Effects](#anc-effects)
+
+  
+
+<a  name="anc-common"></a>
+
+#### Most Common
+
+  
+
+| Asset Type | Prefix | Suffix | Notes |
+
+| ----------------------- | ---------- | ---------- | -------------------------------- |
+
+| Level / Scene | * | | [Should be in a folder called Levels.](#levels) e.g. `Levels/A4_C17_Parking_Garage.unity` |
+
+| Level (Persistent) | | _P | |
+
+| Level (Audio) | | _Audio | |
+
+| Level (Lighting) | | _Lighting | |
+
+| Level (Geometry) | | _Geo | |
+
+| Level (Gameplay) | | _Gameplay | |
+
+| Prefab | | | |
+
+| Material | M_ | | |
+
+| Static Mesh | SM_ | | |
+
+| Skeletal Mesh | SK_ | | |
+
+| Texture | T_ | _? | See [Textures](#anc-textures) |
+
+| Particle System | PS_ | | |
+
+  
+
+<a  name="anc-models"></a>
+
+  
+
+#### 4.2.1a 3D Models (FBX Files)
+
+  
+
+PascalCase
+
+  
+
+| Asset Type | Prefix | Suffix | Notes |
 
-A Value Range only needs to be defined if the bounds of a value are known. While a Slider Range prevents accidental large number inputs, an undefined Value Range allows a user to specify a value outside the Slider Range that may be considered 'dangerous' but still valid.
+| ------------- | ------ | ------ | ----- |
 
-#### Variable Types
+| Characters | CH_ | | |
 
-##### Booleans
+| Vehicles | VH_ | | |
 
-###### Boolean Prefix
-All booleans should be named in PascalCase but prefixed with a verb.
+| Weapons | WP_ | | |
 
-Example: Use `isDead` and `hasItem`, **not** `Dead` and `Item`.
+| Static Mesh | SM_ | | |
 
-###### Boolean Names
-All booleans should be named as descriptive adjectives when possible if representing general information.
+| Skeletal Mesh | SK_ | | |
 
-Try to not use verbs such as `isRunning`. Verbs tend to lead to complex states.
+| Skeleton | SKEL_ | | |
 
-###### Boolean Complex States
-Do not use booleans to represent complex and/or dependent states. This makes state adding and removing complex and no longer easily readable. Use an enumeration instead.
+| Rig | RIG_ | | |
 
-Example: When defining a weapon, do **not** use `isReloading` and `isEquipping` if a weapon can't be both reloading and equipping. Define an enumeration named `WeaponState` and use a variable with this type named `WeaponState` instead. This makes it far easier to add new states to weapons.
+  
 
-##### Enums
-Enums use PascalCase and use singular names for enums and their values. Exception: bit field enums should be plural. Enums can be placed outside the class space to provide global access.
+#### 4.2.1b 3d Models (3ds Max)
 
-Example: 
-```
-public enum WeaponType
-{
-    Knife,
-    Gun
-}
+  
 
-// Enum can have multiple values
-[Flags]
-public enum Dockings
-{
-	None = 0,
```

Comment:
I would leave the section with assigned enum members. These are necessary for some enum usages, such as bitmasks.